examples/Preprocessing_example.md

```diff
@@ -0,0 +1,77 @@
+# Preprocessign an Image
+
+The main objective of the Preprocessing phase is to make as easy as possible for the OCR system to distinguish a character/word from the background. There are several techniques used for Preprocessing but this **Preprocessor** class can do it without you going into details and jargon of traditional methods.
+
+`!pip install -U ocred --no-cache --force-reinstall`
+
+```
+# manually preprocessing an image
+import cv2
+from scipy import ndimage
+from ocred import Preprocessor
+from google.colab.patches import cv2_imshow
+
+
+preprocessed = Preprocessor("/content/images/CosmosOne.jpg")
+```
+
+```
+# scan the image and copy the scanned image
+scanned = preprocessed.scan(inplace=True)
+orig = scanned.copy()
+
+cv2_imshow(orig)
+```
+
+![image](https://user-images.githubusercontent.com/82112540/184899616-2a418f9b-ad82-4bf4-92b7-98a96126dbd1.png)
+
+```
+# remove noise
+noise_free = preprocessed.remove_noise(
+    inplace=True, overriden_image=scanned
+)
+
+cv2_imshow(noise_free)
```

Comment:
Regarding `inplace` (this would make it more clear) -

See how we are setting `inplace=True` and passing in an `overriden_image` named `scanned`. This would edit the actual `scanned` variable, hence if you look at `scanned` after running the function, it would be noise free! This is efficient and this also would not require capturing the value of the returned variable.

On the other hand if you turn `inplace=False`, then `ocred` will generate a copy of `scanned`, remove noise from that copy, and return that copy. This copy can be captured as the return value and the `scanned` variable will not be altered. (not very efficient)

```suggestion
noise_free = preprocessed.remove_noise(overriden_image=scanned)

cv2_imshow(noise_free)
```